PYIC-2484 Core Stubs Support for multiple stubs account

vam-gds opened the pull request:
<!-- Provide a general summary of your changes in the Title above -->
<!-- Include the Jira ticket number in square brackets as prefix, eg `[P4-XXXX] PR Title` -->

## Proposed changes

### What changed
PYIC-2484 Core Stubs Support for multiple stubs account

This adds support for deploying to multiple stubs accounts.

	modified:   deploy/passport/template.yaml

<!-- Describe the changes in detail - the "what"-->

### Why did it change
PYIC-2484 Core Stubs Support for multiple stubs account

This adds support for deploying to multiple stubs accounts.

	modified:   deploy/passport/template.yaml

<!-- Describe the reason these changes were made - the "why" -->

### Issue tracking
<!-- List any related Jira tickets or GitHub issues -->
<!-- List any related ADRs or RFCs -->
<!-- Delete/copy as appropriate -->

- [PYIC-2484](https://govukverify.atlassian.net/browse/PYIC-2484)

## Checklists

### Environment variables or secrets

<!-- Delete if changes DO include new environment variables or secrets -->
- [ ] No environment variables or secrets were added or changed

<!-- Delete if changes DO NOT include new environment variables or secrets -->
- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks


[PYIC-2484]: https://govukverify.atlassian.net/browse/PYIC-2484?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

vam-gds committed:
PYIC-2484 Core Stubs Support for multiple stubs account

This adds support for deploying to multiple stubs accounts.

Fixed typo in comment copy/paste issue.

	modified:   di-ipv-core-stub/deploy/passport/template.yaml

TobySaundersGDS commented on di-ipv-core-stub/deploy/passport/template.yaml:
```diff
@@ -146,10 +146,26 @@ Mappings:
+      desiredTaskCount: 1
+      environment: build
+      platform: stubs
+    "388905755587": #Stubs Build
```
comment here should be prod?

vam-gds committed:
PYIC-2484 Updated github action to deploy to stubs accounts.

Updated github actions to deploy to new stubs accounts.

	modified:   .github/workflows/passport-core-stub.yml
	modified:   di-ipv-core-stub/deploy/passport/template.yaml
